graylog2-web-interface/src/components/permissions/RolesSelector.tsx

```diff
@@ -59,19 +60,19 @@ const _options = (roles, assignedRolesIds, identifier) => roles
   .toArray()
   .map((r) => ({ label: r.name, value: r.name, role: r }));
 
-const _assignRole = (selectedRoleName, roles, onSubmit, setSelectedRoleName, setIsSubmitting, setError) => {
-  if (!selectedRoleName) {
+const _assignRole = (selectedRoleNames, roles, onSubmit, setselectedRoleNames, setIsSubmitting, setError) => {
+  if (!selectedRoleNames) {
     return;
   }
 
-  const selectedRoleNames = selectedRoleName.split(',');
+  const selectedRoleNamess = selectedRoleNames.split(',');
```

Comment:
We should call this `selectedRolesNameArray` for example to make it more readable.